index.d.ts

```diff
@@ -33,6 +50,22 @@ declare namespace del {
 		@default Infinity
 		*/
 		readonly concurrency?: number;
+
+		/**
+		Callback function for progress reporting.
+
+		Called before each file is deleted.
```

Comment:
This says "before", the readme says "after".

Please make sure the docs here are fully in sync.